#goal definition
float32 target_distance
---
#result definition
float32 final_distance
---
#feedback
float32 current_distance